app/Http/Middleware/EnsureEmailIsVerified.php

```diff
@@ -4,22 +4,24 @@
 
 use App\Exceptions\TranslatableException;
 use Closure;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
 
-class EmailVerified
+class EnsureEmailIsVerified
 {
     /**
      * Handle an incoming request.
      *
      * @param \Illuminate\Http\Request $request
-     * @param Closure                  $next
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
      *
-     * @return mixed
+     * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function handle($request, Closure $next)
+    public function handle(Request $request, Closure $next): Response
     {
         if (empty($request->user()?->email_verified_at)) {
             throw new TranslatableException(
-                422,
+                403,
```

Comment:
422 -> The data on the request is invalid
403 -> User does not have permission (this is the usual status for this scenario)